composer.json

```diff
@@ -10,8 +10,18 @@
     }
   ],
   "require": {
-    "php": ">=5.6.0"
+    "php": ">=5.6.0",
```

Comment:
Regarding your previous comments. You can require composer to download your dependencies with a specific php version instead of looking at the version you currently run on your environment.

To do so, you can define a config key (i.e: https://github.com/PrestaShop/autoupgrade/blob/a64e888b5a204610565bd1b55325b0b7655991c7/composer.json#L19-L24):

```json
  "config": {
    "preferred-install": "dist",
    "platform": {
        "php": "5.6"
    }
},
```